### ISSUE TITLE
Async Loading Screen Shows Black Screen on Startup

### ISSUE THREAD

**Issue by NOTz00m** [truong-bui/AsyncLoadingScreen#92]
I'm using the Async Loading Screen plugin in Unreal Engine 5.3.2 and I'm encountering an issue where the screen just shows black on startup (regardless of if i'm playing the game in Standalone version or if it's actually packaged for shipping) and then simply goes to my main menu. I've tried updating my graphics driver, revisiting the setup instructions, looking at fixes for other issues, etc... but the issue persists.

I've tried it in different projects, disabled all other unused plugins, different mp4 files (including the ones provided in the instructions), but it doesn't work for some reason.
Any help would be greatly appreciated, thanks.

**Comment by truong-bui** [comment]
Hi,
Have you carefully followed the [tutorial](https://github.com/truong-bui/AsyncLoadingScreen#tutorial) to understand how it works yet? Usually, this issue is caused by the movie path not being correct.

**Comment by NOTz00m** [comment]
Thanks for the response,

**Comment by NOTz00m** [comment]
My bad, I accdientally closed the issue.
Thank you for the response, but unfortunately. the movie path doesn't seem to be the issue. It stays on the black screen for about 3 seconds and then loads into the main menu. Is there anything I can do about this?

**Comment by truong-bui** [comment]
I recommend following the plugin's tutorial to see how it works. If the movie clip in the tutorial works (which I am sure works fine), there may be something wrong with your clip format or path, or your project has a plugin that conflicts with AsyncLoadingScreen.

**Comment by amirslayer** [comment]
@truong-bui i can verify it, the video is appearing after a long long time!

**Comment by NOTz00m** [comment]
@truong-bui I can say with 100% certainty that I have followed the plugin's tutorial to see how it works and have done the exact steps mentioned there. I made a whole new blank project, enabled ONLY AsyncLoadingScreen (disabled other plugins), followed the steps, put the clip path exactly as mentioned (I used the clip in the tutorial) and it still comes up as just black for 3 seconds and then brings me to the Open Level Button. I also tried the default loading screen setting and that works just fine, however, the startup loading screen does not work. Before adding the movie clip, it does display a black screen with a white border at the bottom, but that doesn't help me considering I need a video at startup.

**Comment by truong-bui** [comment]
Well, it is very weird. Because it just works fine on my side. Here you can check the Async Loading demo that I packaged recently with Unreal version 5.3.2 [https://drive.google.com/file/d/1b7rRsqwR1rl_1TJpnLYUYTImA31SyA3I/view?usp=drive_link](https://drive.google.com/file/d/1b7rRsqwR1rl_1TJpnLYUYTImA31SyA3I/view?usp=drive_link)

Can you please provide a link/screenshot/video to the issue? So I can take a look at it. I cannot help if I can't reproduce the issue on my side.

**Comment by NOTz00m** [comment]
Thank you for the support, I managed to find the issue out through trial and error. Apparently, something to do with the permissions for my Windows Explorer/Account was not allowing Unreal Engine/AsyncLoadingScreen to see the file and therefore was just playing nothing. Atleast for now, I managed to fix it and now it finally works which is great.
Again, thank you for the help and I'm sorry for any inconveniences caused.
